c7n/resources/ecr.py

```diff
@@ -212,6 +219,11 @@ def process(self, resources, event=None):
                             'lifecyclePolicyText', ''))
             except client.exceptions.LifecyclePolicyNotFoundException:
                 r[self.policy_annotation] = {}
+            except ClientError as e:
+                if e.response['Error']['Code'] == 'AccessDeniedException':
+                    self.log.warning('Access Denied on GetLifecyclePolicy for repository: %s' %
+                        r['repositoryName'])
+                    r[self.policy_annotation] = {}
```

Comment:
some changes got pushed to this that now unconditionally and without logging swallow all other exceptions ...